src/components/Main/Main.jsx

```diff
@@ -0,0 +1,252 @@
+                                        </button>
+                                    </div>
+
+                                    <div className="col col-label">
+                                        <button
+                                            className={sortType == "salaryUp" ? "sort-btn up" : sortType == "salaryDown" ? "sort-btn down" : "sort-btn"}
+                                            onClick={() => {
+                                                setSortType(sortType == "salaryUp" ? "salaryDown" : "salaryUp");
+                                                sortTable(sortType == "salaryUp" ? "salaryDown" : "salaryUp");
+                                            }}>
+                                            Salary
+                                            <SortArrows />
+                                        </button>
+                                    </div>
+
+                                    <div className="col col-label">
+                                        <button
+                                            className={sortType == "expUp" ? "sort-btn up" : sortType == "expDown" ? "sort-btn down" : "sort-btn"}
+                                            onClick={() => {
+                                                setSortType(sortType == "expUp" ? "expDown" : "expUp");
+                                                sortTable(sortType == "expUp" ? "expDown" : "expUp");
+                                            }}>
+                                            Years of experience
+                                            <SortArrows />
+                                        </button>
+                                    </div>
+                                </div>
+                                {rows.map((item, idx) => {
+                                    return (
+                                        <div key={idx} className={`row  ${idx}`}>
+                                            <div className="col date">{item["Date"]}</div>
+                                            <div className="col company">{item["Company"] == "" ? '""' : item["Company"]}</div>
+                                            <div className="col position">{item["Position"]}</div>
+                                            <div className="col salary">{item["Salary"]}</div>
+                                            <div className="col experience">{item["Years of experience"] == "" ? '""' : item["Years of experience"]}</div>
+                                        </div>
+                                    )
+                                })}
+                            </div>
+                        </div>
+                    </main>
+                    : <p>No result</p>
+            }
+            <div className="container">
+                <div className="mobile-items-container">
+                    {/* {rows.map((row, idx) => {
```

Comment:
Please remove commented part if unused or add comment why this part is necessary.